products/windows.md

```diff
@@ -259,21 +282,24 @@ releases:
     latest: 6.2.9200
     link: https://learn.microsoft.com/lifecycle/products/windows-8
 
--   releaseCycle: "7 SP1"
+-   releaseCycle: "7-SP1"
+    releaseLabel: "7 SP1"
     releaseDate: 2011-02-22
     support: 2015-01-13
     eol: 2020-01-14
     latest: 6.1.7601
     link: https://learn.microsoft.com/lifecycle/products/windows-7
 
--   releaseCycle: "Vista SP2"
+-   releaseCycle: "6-SP2"
```

Comment:
I preferred to use lowercase for all `releaseCycle` on iphone to avoid uppercase / lowercase confusion, maybe we could do the same here ?

Reply:
Solved with 7a09b7d1df4efe13102d3ed2ad15420ea39799fd